src/features/transcriptions/components/TranscriptionItem.tsx

```
@@ -19,6 +19,17 @@ import {
 import type { TranscriptionRecord } from "../../../types";
 import DotMatrix from "../../../shared/ui/DotMatrix";
 
+const JSON_PREFIX = '{"text":"';
+function stripJsonWrapper(text: string): string {
+  const trimmed = text.trim();
+  if (!trimmed.startsWith(JSON_PREFIX) || !trimmed.endsWith("}")) return text;
+  try {
+    const parsed = JSON.parse(trimmed);
+    return typeof parsed.text === "string" ? parsed.text : text;
+  } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)